module/alicloud_click_house_account: Added support for creating super account. docs: Improved db_cluster_version description. docs: Improved db_node_storage description.

itkillsnone opened the pull request:
resource/alicloud_click_house_account: Added support for creating super account.
docs: Improved db_cluster_version description.
docs: Improved db_node_storage description.

shanye997 commented on variables.tf:
```diff
@@ -122,3 +122,13 @@ variable "account_password" {
+  description = "(Optional, ForceNew) The type of the database account. Valid values: Normal or Super."
+  type        = string
+  default     = "Normal"
+  validation {
```
module 里变量定义对于枚举值不需要增加校验

shanye997 commented on examples/complete/variables.tf:
```diff
@@ -31,4 +31,12 @@ variable "account_password" {
+  description = "(Optional, ForceNew) The type of the database account. Valid values: Normal or Super."
+  type        = string
+  default     = "Normal"
+  validation {
```
同

itkillsnone committed:
resource/alicloud_click_house_account: Added support for creating super account.
docs: Improved db_cluster_version description.
docs: Improved db_node_storage description.
